src/main/java/jp/co/soramitsu/sora/crypto/Consts.java

```diff
@@ -1,14 +1,15 @@
 package jp.co.soramitsu.sora.crypto;
 
-public final class Consts {
+import lombok.experimental.UtilityClass;
 
-  public static final String PROOF_KEY = "proof";
-  public static final String SIGNATURE_KEY = "signature";
+@UtilityClass
+public final class Consts {
```

Comment:
For constants class we can use interface - it will reduce amount of code

Reply:
https://dzone.com/articles/constants-in-java-the-anti-pattern-1